tests/Task.test.ts

```diff
@@ -77,6 +77,48 @@ describe('parsing', () => {
         expect(task!.tags[1]).toEqual('#journal/daily');
     });
 
+    it('parses a task from a line and extract tags and not global filter', () => {
```

Comment:
There's already 3-or-so tests you've added on tag-parsing and there may may well be more...

Suggest moving the tag-related tests you added to a new block specifically on parsing tags, so the code looks something like this:

```typescript
describe('parsing', () => {
    it('parses a task from a line', () => {
        // ...
    });
    // etc
});


describe('parsing tags', () => {
    it('should extract all tags when no global filter', () => {
        // ...
    });
    // etc
});
```

Amongst other benefits, this will help our future selves understand the structure of the tests, and make it easy to run just the tag tests during development....

Reply:
Oh, in case this snippet is confusing, I was experimenting with renaming the new tests to begin with ‘should….’ - so the second block in the snippet above is intended to be your recent tests. 

Background: It can be a challenge to name tests so that when/if they fail, it is clear what went wrong. 

Someone once told me that if one tries to name every test beginning with ‘should’, it makes it easier to come up with words to express the intention. I’ve found it works well.

Reply:
Covered as part of test refactor.